examples/ex9p.cpp

```diff
@@ -52,28 +52,35 @@ double inflow_function(const Vector &x);
 // Mesh bounding box
 Vector bb_min, bb_max;
 
+// Fem discrization
+// (continous or discontinous galerkin)
+enum fem_disc {CG_FE, DG_FE};
 
-/** A time-dependent operator for the right-hand side of the ODE. The DG weak
+/** A time-dependent operator for the right-hand side of the ODE. The weak
     form of du/dt = -v.grad(u) is M du/dt = K u + b, where M and K are the mass
     and advection matrices, and b describes the flow on the boundary. This can
     be written as a general ODE, du/dt = M^{-1} (K u + b), and this class is
     used to evaluate the right-hand side. */
 class FE_Evolution : public TimeDependentOperator
 {
 private:
-   HypreParMatrix &M, &K;
-   const Vector &b;
+   ParBilinearForm &M, &K;
+   ParLinearForm &b;
+   HypreParMatrix *M_mat, *K_mat;
+   HypreParVector *B_vec;
    HypreSmoother M_prec;
    CGSolver M_solver;
+   fem_disc fem_type;
```

Comment:
Seems that fem_type is not used in the class.

Reply:
fixed :+1: